CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 10.8.0
+
```

Comment:
Update the changelog?

Reply:
Going to have to manually do this I think because travis didn't auto-generate one 🤔